### ISSUE TITLE
Can we turn off the "Get more info for this Issue" prompt

### ISSUE THREAD

**Issue by gep13** [octobox/octobox#1133]
### What feature you'd like to see

I would like the ability to turn off the reminder/prompt to install the GitHub application:

![image](https://user-images.githubusercontent.com/1271146/47240536-80182080-d3e0-11e8-8fe1-9f99e3adbf1d.png)

## Motivation

I have already installed the GitHub App for all the repositories that I can, as there are some that I am not able to.  I don't need reminded about the GitHub Application, as I am already using, and loving it.

## Pitch

Simply put, it is a little bit annoying to see the reminder/prompt, when I already have it installed, and running, on other repositories.

**Comment by hakusaro** [comment]
Not only that, but it's worth killing off the "install the app" banner in the user bar too.

**Comment by hakusaro** [comment]
I sent a wip pr for this issue in #1136. If I've gone in the right direction I'll add some tests and see if I can get this upstreamed rapidly. :)

**Comment by andrew** [comment]
I'd prefer to avoid adding more check boxes to the settings page.

I think we also need to take a different approach to solving https://github.com/octobox/octobox/issues/1133, just turning off all prompts doesn't really work to help get Octobox.io to become a sustainable open source project, which the GitHub app will be the main source of income.

The way I'd like to change the behavior of the popup would be to turn it from hover to click if the user has installed the GitHub app on any one or more repository that they have access to.

**Comment by andrew** [comment]
https://github.com/octobox/octobox/pull/1183 will hide the message in the header once a user has authorized the GitHub app, even if they haven't installed it on any of their repos
